client/src/components/graph/graph.js

```diff
@@ -3,43 +3,24 @@ import React from "react";
 import * as d3 from "d3";
 import { connect } from "react-redux";
 import mat4 from "gl-mat4";
-import vec3 from "gl-vec3";
 import _regl from "regl";
 import memoize from "memoize-one";
-import {
-  Button,
-  AnchorButton,
-  Tooltip,
-  Popover,
-  Menu,
-  MenuItem,
-  Position,
-  NumericInput,
-  Icon,
-  RadioGroup,
-  Radio
-} from "@blueprintjs/core";
 
 import * as globals from "../../globals";
 import setupSVGandBrushElements from "./setupSVGandBrush";
-import actions from "../../actions";
 import _camera from "../../util/camera";
 import _drawPoints from "./drawPointsRegl";
 import scaleLinear from "../../util/scaleLinear";
-import { World } from "../../util/stateManager";
 
 /* https://bl.ocks.org/mbostock/9078690 - quadtree for onClick / hover selections */
 
 @connect(state => ({
```

Comment:
there are unused properties, eg, `libraryVersions`.  Please remove so that they do not trigger unnecessary updates.

Reply:
and please make sure you add them to the menu bar, which needs them :-)